src/main/java/com/uber/profiling/Arguments.java

```diff
@@ -71,7 +73,9 @@
     private String configFile;
 
     private String appIdVariable;
+    private String appNameVariable;
     private String appIdRegex = DEFAULT_APP_ID_REGEX;
+    private String appNameRegex = DEFAULT_APP_ID_REGEX;
```

Comment:
@acharneski - This was the reason app_id and app_name were returning the same name.  I have made changes and tested it.